docs/diagrams/make_diagrams.py

```
@@ -0,0 +1,177 @@
+  2. Three engine components + L2 three-program model
+  3. Rust-suitability map over the architecture
+Run: python3 make_diagrams.py  ->  architecture.svg
+Pure stdlib; no external deps. Edit here and re-run to regenerate.
+"""
+import html, os
+
+W = 1180
+PADDED = []  # (svg fragment, height) appended per panel
+
+# ---- palette ----
+BG = "#0e1116"; PANEL = "#161b22"; INK = "#e6edf3"; MUTE = "#8b949e"
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)